app/assets/stylesheets/mixins.scss

```diff
@@ -1,3 +1,33 @@
+/* Responsive mixin
```

Comment:
We need to include the mixins, from legacy file `populities`, that we're using. Quite possible, this file won't suffer future modifications, once all the mixins are included.